extract-files.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright (C) 2010 The Android Open Source Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DEVICE=seagull
+MANUFACTURER=sony
+STOCK="aosp_d5103-userdebug 5.1.1 LMY48G eng.root.20150718.041511 test-keys"
```

Comment:
The version BELOW is the highest one available on Sony AOSP pages (18.1.A.2.25). Which correspond to a Android 5.1. In order to extract the needed files (using this script), I had to flash the device using an AOSP 5.1 package available on FXP. The build version of this package was the one above. Should I comment line 19 and uncomment line 20 ?